public/data/luvgadhvi.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Luv Gadhvi",
+  "bio": "Full Stack Developer",
+  "avatar": "https://github.com/luvgadhvi.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/luvgadhvi",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/luv-gadhvi-08b198a1/",
+      "icon": "linkedin",
+      "isLinkIcon": true,
```

Comment:
I think these lines should not be required Please check how other socials are implemented and try to make it the same way

Reply:
This line is required, Check the Links.js file, I have put conditional rendering code in case that logo is not provided by prime icon libary.